src/materials/SteelCreepDamageOh.C

```diff
@@ -117,7 +124,13 @@ SteelCreepDamageOhTempl<is_ad>::updateQpDamageIndex()
     return;
   }
 
-  GenericRankTwoTensor<is_ad> creep_increment = _creep_strain[_qp] - _creep_strain_old[_qp];
+  _combined_creep_strain[_qp].zero();
+  for (unsigned int i = 0; i < _creep_strain_names.size(); ++i)
+  {
+    _combined_creep_strain[_qp] += (*_creep_model[i])[_qp];
+  }
```

Comment:
For scopes of one line, we typically remove the braces and rely on indentation to read the code, e.g.:

``` 
for (unsigned int i = 0; i < _creep_strain_names.size(); ++i) 
  _combined_creep_strain[_qp] += (*_creep_model[i])[_qp];
  ```